build.cake

```diff
@@ -1,3 +1,6 @@
+#tool "KuduSync.NET"
```

Comment:
Recommendation here would be to pin tool version, example:
```
#tool nuget:?package=Cake.Foo&version=1.2.3
```
https://cakebuild.net/docs/fundamentals/preprocessor-directives